server/src/main/java/org/elasticsearch/index/query/functionscore/DecayFunctionBuilder.java

```diff
@@ -210,6 +210,7 @@ private AbstractDistanceScoreFunction parseVariable(String fieldName, XContentPa
 
         // dates and time and geo need special handling
         parser.nextToken();
+        // TODO these ain't gonna work with runtime fields
```

Comment:
interesting scenario that I never though about: scoring based on runtime fields. You already added this to the runtime fields meta issue right?

Reply:
Yeah, I added it.